README.md

```diff
@@ -1,35 +1,37 @@
 Passport-verify-stub-relying-party
 ==================================
 
-GOV.UK Verify provides a federation of Identity Providers that Government services can use to verify the identity of their users.
+Passport-verify-stub-relying-party is a Node.js application built to demonstrate how a government service connecting to GOV.UK Verify can build a Node.js service that uses the [Verify Service Provider](https://github.com/alphagov/verify-service-provider) to communicate with the Verify Hub. 
 
-This is an application that has been built to demonstrate how a Relying Party (Government Service) can build a web service using node.js, that uses the [Verify Service Provider](...) microservice to communicate with
-GOV.UK Verify's Hub.
+This application uses the Express framework and the [Passport](https://www.npmjs.com/package/passport) identification middleware.
```

Comment:
We should note that it uses the passport-verify package that we publish as well

Reply:
agreed, removed reference to pass, and explained what defaults are used instead.  There is still reference to the paas flag in the package.json scripts but we can fix as a separate story

Reply:
This dependency is already listed in the bullet points below this section.